src/popup/router/components/Modals/QrCodeReader.vue

```diff
@@ -3,38 +3,59 @@
     v-if="browserReader || !cameraAllowed"
     has-close-button
     class="qr-code-reader"
+    centered
+    from-bottom
     @close="resolve"
+    bodyClass="body-extend"
   >
     <template slot="header">
-      <QrScan class="icon" />
-      {{ title }}
+      <div class="qr-scan-wrapper">
+        <QrScan />
+      </div>
+      <span v-if="cameraAllowed">{{ title }}</span>
+      <span v-else>
+        {{ $t('modals.qrCodeReader.grantPermission') }}
+        <div class="subtitle">{{ $t('modals.qrCodeReader.subtitle') }}</div>
+      </span>
     </template>
 
-    <div class="camera">
-      <div v-show="cameraAllowed">
-        <video
-          v-show="cameraAllowed"
-          ref="qrCodeVideo"
-        />
-      </div>
-      <div v-if="!cameraAllowed">
-        {{ $t('modals.qrCodeReader.cameraNotAllowed') }}
+    <template slot="customBody" >
```

Comment:
Please use short version of slot definitions.
```suggestion
    <template #customBody>
```